packages/firebase/index.d.ts

```diff
@@ -6949,15 +6949,56 @@ declare namespace firebase.messaging {
    *
    */
   interface Messaging {
+    /**
+     * Deletes the registration token associated with this messaging instance and unsubscribes
+     * this messaging instance from the push subscription.
+     *
+     * @return The promise resolves when the token has been successfully deleted.
+     */
+    deleteToken(): Promise<boolean>;
+
     /**
      * To forcibly stop a registration token from being used, delete it
      * by calling this method.
      *
      * @param token The token to delete.
      * @return The promise resolves when the token has been
      *   successfully deleted.
+     *
+     * @deprecated Use deleteToken() instead.
      */
     deleteToken(token: string): Promise<boolean>;
+
+    /**
+     * Subscribes the messaging instance to push notifications. Returns an FCM registration
+     * token that can be used to send push messages to the user.
+     *
+     * If notification permission isn't already granted, this method asks the
+     * user for permission. The returned promise rejects if the user does not
+     * allow the app to show notifications.
+     *
+     * @param options.vapidKey the public server key provided to push services. It is used to
+     * authenticate the push subscribers to receive push messages only from sending servers that
+     * holds the corresponding private key. If it is not provided, a default VAPID key will be used.
+     * Note that some push services (Chrome Push Service) require a non-default VAPID key.
+     * Therefore, it is recommended to generate and import a VAPID key for your project with
+     * {@link https://firebase.google.com/docs/cloud-messaging/js/client#configure_web_credentials_with_fcm Configure Web Credentials with FCM}.
+     * See {@link https://developers.google.com/web/fundamentals/push-notifications/web-push-protocol The Web Push Protocol}
+     * for details on web push services.}
+     *
+     * @param options.serviceWorkerRegistration the service worker registration for receiving push messaging.
+     * If it is not provided explicitly, you need to have a `firebase-messaging-sw.js` at your root location.
+     * See {@link https://firebase.google.com/docs/cloud-messaging/js/client#retrieve-the-current-registration-token Retrieve the current registration token}
+     * for more details.
+     *
+     * @return The promise resolves with the FCM token string.
```

Comment:
Is the "FCM token string" the registration token?

It might be confusing in this context, but we prefer "registration token" as described here.  Your call.

https://firebase.google.com/docs/cloud-messaging/concept-options#credentials

Reply:
It is referring the same subject. I think keeping it consistence across the board make sense. Changing accordingly.